File: CS/Q212065/DrillDownControl.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Drawing;
using System.Data;
using System.Text;
using System.Windows.Forms;

namespace Q212065 {
    public partial class DrillDownControl : UserControl {
        public DrillDownControl() {
            InitializeComponent();
        }

        public object DataSource {
            get { return gridControl1.DataSource; }
            set { gridControl1.DataSource = value; }
        }
    }
}

File: VB/Q212065/DrillDownControl.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Drawing
Imports System.Data
Imports System.Text
Imports System.Windows.Forms

Namespace Q212065
	Partial Public Class DrillDownControl
		Inherits UserControl
		Public Sub New()
			InitializeComponent()
		End Sub

		Public Property DataSource() As Object
			Get
				Return gridControl1.DataSource
			End Get
			Set(ByVal value As Object)
				gridControl1.DataSource = value
			End Set
		End Property
	End Class
End Namespace
